pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -233,6 +235,34 @@ func (m *kubeGenericRuntimeManager) generateLinuxContainerConfig(container *v1.C
 		SecurityContext: m.determineEffectiveSecurityContext(pod, container, uid, username),
 	}
 
+	if utilfeature.DefaultFeatureGate.Enabled(features.SupportUlimits) {
+		lc.Resources.Ulimits = make(map[string]*runtimeapi.LimitValue)
+		for key, value := range container.Resources.Requests {
+			if strings.HasPrefix(string(key), "ulimit-") {
```

Comment:
How about using a prefix like `ulimits.k8s.io/`? I think it's more consisctent with other resources.

Reply:
i tried that in a previous iteration and ran into trouble. will dig into my notes and post back